### DISCUSSION TITLE
Customization

### DISCUSSION THREAD

**Discussion by EnzoBustos** [anuraghazra/github-readme-stats#3631] [Q&A] [answered]
Hey, I'm a bit new to all this, but my question is: 

- How I change the language of the Stats Card? I want it to display as pt-br?
- How do I put spaces in title? I want it to display like Enzo Bustos's GitHub Stats
- How do I create a custom palette for the circle, title, background, etc?

**Comment by qwerty541** [comment, marked as answer]
Hey, @EnzoBustos! Here is the answers on your questions.

### Locale

You can choose locale of your card using `&locale=pt-br` query string option. Here is an example:

<img src="https://github-readme-stats.vercel.app/api?username=enzobustos&locale=pt-br" />

```html
<img src="https://github-readme-stats.vercel.app/api?username=enzobustos&locale=pt-br" />
```

### Title

You can specify custom title for your card using `&custom_title=Enzo%20Bustos%27s%20GitHub%20Stats` query string option. Here is an example:

<img src="https://github-readme-stats.vercel.app/api?username=enzobustos&custom_title=Enzo%20Bustos%27s%20GitHub%20Stats" />

```html
<img src="https://github-readme-stats.vercel.app/api?username=enzobustos&custom_title=Enzo%20Bustos%27s%20GitHub%20Stats" />
```

### Custom pallete

Here is the list of query string options for creating custom pallete:

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/d1534590-088d-4f73-93cc-0a29e4a8e5ec)

For your information, you could easily find out all this yourself if you carefully read the documentation.
